siliconcompiler/report/streamlit_viewer.py

```diff
@@ -568,7 +570,20 @@ def show_title_and_runs(title_col_width=0.7):
         streamlit.columns([title_col_width, 1 - title_col_width], gap="large")
 
     with title_col:
-        streamlit.title(f'{new_chip.design} dashboard', anchor=False)
+        # in case ui_width is 0
+        if ui_width <= 0:
+            icon_width = 0.01
+            streamlit.title(f'{new_chip.design} dashboard', anchor=False)
+        else:
+            # 77 because you should resize by a factor of 0.5 - original pixel
+            # width is 308
+            icon_width = 77 / (ui_width * title_col_width)
```

Comment:
would it be better to open the image and get the number of pixels? incase it changes? we already open it to pass it in as an icon, so we can just use that again.